ingest-app/README.md

```diff
@@ -98,6 +98,7 @@ The providers that exist in the CMR are administered through the Ingest API. A p
  * `short-name` - A unique identifier of the provider. It is similar to `provider-id`, but more descriptive. It allows spaces and other special characters. The maximum length of `short-name` is 128 characters. `short-name` defaults to `provider-id`.
  * `cmr-only` - True or false value that indicates if this is a provider that ingests directly through the CMR Ingest API or the legacy ECHO Catalog REST Ingest API. A CMR Only provider will still have ACLs configured in ECHO and support ordering through ECHO. A CMR Only provider may even still have data in Catalog REST but it will not be kept in sync with the CMR. `cmr-only` defaults to false.
  * `small` - True or false value that indicates if this is a provider that has a small amount of data and its collections and granules will be ingested into the `SMALL_PROV` tables. `small` defaults to false.
+ * `consortiums` - A string of space delimited projects/efforts/tags(Example: CWIC GEOSS FEDEO) that a provider wants their collections automatically indexed with. Each project/effort/tag can only contain alphanumeric characters and underscores. Invalid consortiums is rejected at ingest. 
```

Comment:
```suggestion
 * `consortiums` - A string of space delimited consortiums. Consortiums are projects, efforts or tags that a provider is associated with. Consortium can only contain alphanumeric characters and underscores. For example: `CWIC EOSDIS GEOSS`.
```

Reply:
We also need to indicate if `consortiums` is a mandatory field or optional field here. I think it should be an optional field.

Reply:
Done.